src/mmw/sass/components/_tabs.scss

```diff
@@ -57,6 +57,13 @@
     }
 }
 
+.data-catalog-stage-results-tab-pane {
+    .tab-content {
+      // Set a specific height to allow y-scroll:
+      height: calc(100% - 53px) !important;
```

Comment:
This is likely causing the grey box issues, since it may double the work here: https://github.com/WikiWatershed/model-my-watershed/blob/88a933ae2126630bf303cd83c5ecf3bd84857852/src/mmw/sass/pages/_data-catalog.scss#L25